Lib/asyncio/sslproto.py

```diff
@@ -636,11 +633,12 @@ def _process_write_backlog(self):
                         self._on_handshake_complete)
                     offset = 1
                 else:
-                    ssldata = self._sslpipe.shutdown(self._finalize)
+                    ssldata = self._sslpipe.shutdown()
```

Comment:
Do we want to wrap this in try..finally to guarantee that `self._finalize` will be called?